docs/proposals/scheduler/controller.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+
+
+// ------------------
+// Scheduling
+// ------------------
+
+message SchedulingRequest {
+    // definition of the workload
```

Comment:
Here you would have a `Workload`, as in the definition of an `orka` workload.

Reply:
[Done](https://github.com/dev-sys-do/orka/pull/2/commits/dae37730c6d56acaa1cfbe39fe77b81d9714271b#diff-de3632c29b02edb1da9170fa6071371537f5980fb145bb4c842e69e2094a41c9R8-R29)! It's pretty much the same thing as the definition from the agent.